content/en/developers/integrations/python.md

```diff
@@ -78,110 +68,98 @@ For Linux installations, avoid modifying your system Python. Datadog recommends
 
 {{< /tabs >}}
 
-## Install pipx
+## Install developer tooling
+
+You have 2 options to install the `ddev` CLI.
 
-The `pipx` python package is required for the `ddev` command line tools.
+
+### Install using a GUI
 
 {{< tabs >}}
 {{% tab "MacOS" %}}
-1. Install pipx:
-   ```
-   brew install pipx
-   ```
-1. Check the Homebrew installation output and run any additional commands recommended by the installation script.
-
-1. Verify that pipx is installed:
+1. In your browser, download the `.pkg` file: [ddev-5.2.1.pkg](https://github.com/DataDog/integrations-core/releases/download/ddev-v5.2.1/ddev-5.2.1.pkg)
```

Comment:
If the version is not auto updated, let's add a note stating to use the required version of ddev and 5.2.1 is provided as an example in all the referenced places.

Reply:
The version will be dynamically rendered in an upcoming PR